src/lang/danish.txt

```diff
@@ -924,6 +924,7 @@ STR_GAME_OPTIONS_CURRENCY_ZAR                                   :Sydafrikansk Ra
 STR_GAME_OPTIONS_CURRENCY_CUSTOM                                :Brugerdefineret...
 STR_GAME_OPTIONS_CURRENCY_GEL                                   :Georgiske Lari (GEL)
 STR_GAME_OPTIONS_CURRENCY_IRR                                   :Iranske Rialer (IRR)
+STR_GAME_OPTIONS_CURRENCY_MXN                                   :Mexikanske Peso (MXN)
```

Comment:
As helpful as it is, we like all non-english.txt translations to go via the webtranslator rather than being added via PRs. Can you remove all of these?

Reply:
oh okay, sure.